packages/sanity/src/desk/panes/document/timeline/timelineMenu.tsx

```diff
@@ -136,10 +138,12 @@ export function TimelineMenu({chunk, mode, placement}: TimelineMenuProps) {
   const timeLabel = useFormattedTimestamp(chunk?.endTimestamp || '')
 
   const revLabel = chunk
-    ? `${upperFirst(formatTimelineEventLabel(chunk.type))}: ${timeLabel}`
-    : 'Latest version'
+    ? `${upperFirst(t(`desk.timeline.${chunk.type}`))}: ${timeLabel}`
+    : t('desk.timeline.latest-version')
 
-  const sinceLabel = chunk ? `Since: ${timeLabel}` : 'Since: unknown version'
+  const sinceLabel = chunk
+    ? t('desk.timeline.since', {timeLabel: timeLabel})
+    : t('desk.timeline.since-version-missing')
```

Comment:
These ones are somewhat complicated. Ideally we'll want to remove the  `useFormattedTimestamp`  here and use the [`datetime` formatter](https://www.i18next.com/translation-function/formatting#datetime) (based on `Intl.DateTimeFormat`).

Something like:
```ts
const sinceLabel = chunk
  ? t('timeline.since', {
      timestamp,
      formatParams: {timestamp: {dateStyle: 'medium', timeStyle: 'short'}},
    })
  : t('timeline.since-version-missing')
```
And for the locale resource:
```ts
const resources = {
  // ...
  'timeline.since': 'Since: {{timestamp, datetime}}'
}
```

This will format the date according to the current locale and calendar, eg in US English it would give `Aug 22, 2021, 1:15 AM` while in Norwegian it would give `22. aug. 2021, 01:15`, and in Thai `22 ส.ค. 2564 01:15`.